src/utils/RouteUtilsV3.js

```diff
@@ -0,0 +1,310 @@
+// @flow
+import AllStopUtils from './AllStopUtils';
+import GraphhopperUtils from './GraphhopperUtils';
+import LogUtils from './LogUtils';
+import ParseRouteUtils from './ParseRouteUtilsV3';
+
+/**
+ * Returns the flattened version of arr.
+ *
+ * @param arr
+ * @returns {Array<Object>}
+ */
+function flatten(arr: Array<Array<Object>>): Array<Object> {
+  return [].concat(...arr);
+}
+
+/**
+ * Returns whether or not location is a bus stop.
+ *
+ * @param location
+ * @returns {Promise<boolean>}
+ */
+async function isBusStop(location: string): Promise<boolean> {
+  const stops = await AllStopUtils.fetchAllStops();
+  return stops.filter(s => s.name === location).length > 0;
+}
+
+/**
+ * Returns whether [route] contains a bus trasnfer
```

Comment:
transfer

Reply:
hehe tyty for all these helpful comments lol i didnt even bother fixing the old stuff xD